.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
I didn't know about this setting. I'll add this to the various Sanctuary projects. :)